core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/jupyter/Integration.kt

```diff
@@ -56,7 +35,7 @@ import kotlin.reflect.KProperty
 import kotlin.reflect.full.isSubtypeOf
 
 /** Users will get an error if their Kotlin Jupyter kernel is older than this version. */
-private const val MIN_KERNEL_VERSION = "0.11.0.198"
+private const val MIN_KERNEL_VERSION = "0.11.0.311"
```

Comment:
We only have to update min kernel version if compatibility is broken, i.e. new version of the dataframe won't work with old kernels. Is it broken now? I see that render produces a different result than before. Also, did you check how everything works in Jupyter notebooks?